sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1309,6 +1309,13 @@ object CodeGenerator extends Logging {
   // bytecode instruction
   final val MUTABLESTATEARRAY_SIZE_LIMIT = 32768
 
+  // Display the total compile time for generating java code in the output of the test case.
```

Comment:
Although this is designed for a test case, in this context, this is a general accumulator in `CodeGenerator`. Could you revise into a more proper description?
cc @gatorsmile

Reply:
OK. I will supplement a more proper description.